src/utils/daos.c

```diff
@@ -945,80 +964,87 @@ cont_op_hdlr(struct cmd_args_s *ap)
 	assert(ap != NULL);
 	op = ap->c_op;
 
-	/* All container operations require a pool handle, connect here.
-	 * Take specified pool UUID or look up through unified namespace.
-	 */
-	if ((op != CONT_CREATE) && (ap->path != NULL)) {
-		struct duns_attr_t dattr = {0};
-		struct dfuse_il_reply il_reply = {0};
-
-		ARGS_VERIFY_PATH_NON_CREATE(ap, out, rc = RC_PRINT_HELP);
-
-		/* Resolve pool, container UUIDs from path if needed
-		 *
-		 * Firtly check for a unified namespace entry point, then if
-		 * that isn't detected then check for dfuse backing the
-		 * path, and print pool/container/oid for the path.
+	/* cont_copy uses its own src and dst variables, and does
+	 * not use the regular pool and cont ones stored in the
+	 * ap struct. This is because for a copy it is necessary
+	 * to explicity specify whether a container is a src or
+	 * dst */
+	if (op != CONT_COPY) {
+		/* All container operations require a pool handle, connect here.
+		 * Take specified pool UUID or look up through unified namespace.
 		 */
-		rc = duns_resolve_path(ap->path, &dattr);
-		if (rc) {
+		if ((op != CONT_CREATE) && (ap->path != NULL)) {
+			struct duns_attr_t dattr = {0};
+			struct dfuse_il_reply il_reply = {0};
 
-			rc = call_dfuse_ioctl(ap->path, &il_reply);
-			if (rc != 0) {
-				fprintf(stderr, "could not resolve pool, "
-					"container by path: %d %s %s\n",
-					rc, strerror(rc), ap->path);
+			ARGS_VERIFY_PATH_NON_CREATE(ap, out, rc = RC_PRINT_HELP);
 
-				D_GOTO(out, rc);
+			/* Resolve pool, container UUIDs from path if needed
+		 	*
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs